draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -318,10 +331,12 @@ group.<unique-2>.zones.$CATZ  0 IN TXT    operator-y-nsec3
 
 ```
 
-The catalog zone (snippet) above is an example where the producer signals how the consumer(s) shall treat DNSSEC for the zones "example.net." and "example.com."
-
-For "example.com.", the consumer might be implemented and configured in the way that the member zone will not be signed with DNSSEC.
-For "example.net.", the consumers, at two different operators, might be implemented and configured in the way that the member zone will be signed with a NSEC3 chain.
+In this scenario, consumer(s) shall not sign the member zone "example.com." with
+DNSSEC.
+For "example.net.", the consumers, at two different operators, shall configure
+the member zone to be signed with a NSEC3 chain.  The group value that indicates
```

Comment:
```suggestion
the member zone to be signed with an NSEC3 chain.  The group value that indicates
```

(also, would like to echo Libor's sentiment that non-wrapped sentences are way more fun in diffs)